apps/auth/README.md

```diff
@@ -0,0 +1,19 @@
+# Auth
+
+## Installation
```

Comment:
We should probably remove the boilerplate

Reply:
okay